ur_kinematics/CMakeLists.txt

```
@@ -152,14 +176,22 @@ install(TARGETS
     ur5e_kin
     ur10_kin
     ur10e_kin
+    ur16e_kin
+    ur8long_kin
+    ur15_kin
+    ur18_kin
+    ur20_kin
+    ur20_kin
```

```suggestion
    ur30_kin
```
Should this be ur30 instead of ur20?